common/core/src/iosMain/kotlin/com/trafi/locations/LocationsApi.kt

```diff
@@ -0,0 +1,28 @@
+@file:Suppress("unused", "EXTENSION_SHADOWED_BY_MEMBER")
```

Comment:
What does these suppress does exactly ? 🤗

Reply:
They suppress warnings from the Kotlin compiler. So they're not necessary, but without suppressing them the compiler & IDE will display warnings such as
- `unused` - "This function is not used"
- `EXTENSION_SHADOWED_BY_MEMBER` - "This extension function is shadowed by a member function of the same name"

The first is irrelevant - we know these functions are not used within the Kotlin code, but they are used from Swift in the published xcframework.
The second is also not a problem - in Kotlin e.g. `suspend fun UsersApi.createOrGetUser()` and `fun UsersApi.createOrGetUser(): CFlow` may clash, but when translated to Objective-C / Swift their signatures are different - `func createOrGetUser(completionHandler: ..)` and `func createOrGetUser(): CFlow`.